src/test/java/org/apache/maven/plugins/shade/resource/ServiceResourceTransformerTest.java

```diff
@@ -85,6 +85,49 @@ public void relocatedClasses() throws Exception {
             tempJar.delete();
         }
     }
+
+    @Test
+    public void mergeRelocatedFiles() throws Exception {
+        SimpleRelocator relocator =
+                new SimpleRelocator( "org.foo", "borg.foo", null, Arrays.asList( "org.foo.exclude.*" ) );
+        relocators.add( relocator );
+
+        String content = "org.foo.Service\norg.foo.exclude.OtherService\n";
```

Comment:
Why not use constant `NEWLINE` as is used later

Reply:
Make sense. I just copy from the test above. Code refactoring can be another topic that I don't mess up one PR. But it's fair enough for new code :)

Reply:
Resolved.